toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -237,19 +237,22 @@ func createAllSRPMs(specsDir, distTag, buildDir, outDir string, workers, concurr
 	timestamp.StartEvent("packing SRPMS", nil)
 	defer timestamp.StopEvent(nil)
 
+	rootCtx, cancelFunc := context.WithCancel(context.Background())
```

Comment:
Is this needed? You don't seem to be using `cancelFunc` here.

Reply:
Probably don't even need a root ctx here at all, each function can handle their own ctx